mapper-javabean/src/main/java/org/hibernate/search/v6poc/entity/javabean/log/impl/Log.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.search.v6poc.entity.javabean.log.impl;
+
+import org.hibernate.search.v6poc.entity.pojo.logging.spi.PojoTypeModelFormatter;
+import org.hibernate.search.v6poc.entity.pojo.model.spi.PojoRawTypeModel;
+import org.hibernate.search.v6poc.util.SearchException;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.annotations.FormatWith;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "HSEARCH-JAVABEAN")
+public interface Log extends BasicLogger {
+
+	@Message(id = 1, value = "Unable to find property '%2$s' on type '%1$s'")
+	SearchException cannotFindProperty(@FormatWith(PojoTypeModelFormatter.class) PojoRawTypeModel<?> typeModel,
+			String propertyName);
+
+	@Message(id = 2, value = "Cannot read property '%2$s' on type '%1$s'")
```

Comment:
Until now, you were adding dots at the end of the log messages? Feel free to ignore this comment though.

Reply:
Done